packages/function-memoize-one/index.js

```diff
@@ -0,0 +1,44 @@
+module.exports = memoizeOne;
+
+function defaultEqual(arr1, arr2) {
```

Comment:
why not just use `JSON.stringify(arr1) === JSON.stringify(arr2)`? I think the result would be the same.

Reply:
you are right that the result will be the same, however the execution here is slower with `JSON.stringify`.